src/goSuggest.ts

```diff
@@ -50,8 +50,22 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				let lineTillCurrentPosition = lineText.substr(0, position.character);
 				let autocompleteUnimportedPackages = config['autocompleteUnimportedPackages'] === true && !lineText.match(/^(\s)*(import|package)(\s)+/);
 
-				if (lineText.match(/^\s*\/\//)) {
-					return resolve([]);
+				if (lineText.match(/\//)) {  // complete exported item`s comment
+					if (!lineText.match(/^\/\//)) {
+						return resolve([]);
+					}
+					let nextLineText = document.lineAt(position.line + 1).text;
+					if (nextLineText) {
+						let tokens = nextLineText.split(' ');
+						if (tokens.length < 2 || ['type', 'func'].indexOf(tokens[0]) < 0 || tokens[1].match(/^[a-z]/)) {
+							return resolve([]);
+						}
+						let kind = tokens[0];
+						let symbol = tokens[1].split('(')[0];
```

Comment:
This will fail to find method names with receivers. For example: `func (s *Server) poll() `

Reply:
I think the function`s comments only using to completing the named functions😂, it is shot of tests. Thinks your reviews, i will make it pretty.